File: stapp/app/src/main/java/com/xvesa/stapp/MainActivity.java
package com.xvesa.stapp;

import android.os.Bundle;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    private NavigationHandler h;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        this.setContentView(R.layout.activity_main);
        h = new NavigationHandler(this);
        h.onCreateHandle();
    }

    @Override
    public void onEnterAnimationComplete() {
        super.onEnterAnimationComplete();
        h.afterCreateHandle();
    }
}

File: stapp/app/src/main/java/com/xvesa/stapp/MainActivity.kt
package com.xvesa.stapp

import android.os.Bundle
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private var handler: NavigationHandler? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        this.enableEdgeToEdge()
        this.setContentView(R.layout.activity_main)
        handler = NavigationHandler(this)
        handler!!.onCreateHandle()
    }

    override fun onEnterAnimationComplete() {
        super.onEnterAnimationComplete()
        handler!!.afterCreateHandle()
    }
}